docs/ai/includes/vector-databases.md

```diff
@@ -5,7 +5,9 @@ ms.date: 07/29/2024
 ms.topic: include
 ---
 
-AI applications often use data vector databases and services to improve relevancy and provide customized functionality. Many of these services provide a native SDK for .NET, which you can use directly. Semantic Kernel provides an extensible component model that enables you to use different vector stores without needing to learn each SDK. Semantic Kernel provides connectors for the following vector databases and services:
+AI applications often use data vector databases and services to improve relevancy and provide customized functionality. Many of these services provide a native SDK for .NET, while others offer a REST service you can connect to through custom code. Semantic Kernel provides an extensible component model that enables you to use different vector stores without needing to learn each SDK.
+
+Semantic Kernel provides connectors for the following vector databases and services:
 
 - [Vector Database in Azure Cosmos DB for NoSQL](https://github.com/microsoft/semantic-kernel/tree/main/dotnet/src/Connectors/Connectors.Memory.AzureCosmosDBNoSQL)
```

Comment:
Suggest alphabetizing the list as well. The Cosmos DB scenarios probably aren't the most interesting to our customers, so they also don't make sense at the top of the list.